df.legends.xml

```diff
@@ -104,15 +104,15 @@
 
         <pointer name='item' type-name='item'/>
 
-        <int32_t since='v0.34.01'/>
-        <int32_t since='v0.34.01'/>
-        <int32_t since='v0.34.01'/>
-        <int32_t since='v0.44.01'/>
-        <int32_t since='v0.44.01'/>
-        <int32_t since='v0.44.01'/>
-        <int32_t since='v0.44.01'/>
-        <int32_t since='v0.44.01'/>
-        <int32_t since='v0.44.04'/>
+        <int32_t name='abs_tile_x'/>
```

Comment:
I'd keep the `since` attributes since they're still potentially meaningful, but I also don't know if anyone relies on them heavily enough for it to matter.